docker/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM ubuntu:20.04
+RUN \
+  # configure the "jhipster" user
+  groupadd jhipster && \
+  useradd jhipster -s /bin/bash -m -g jhipster -G sudo && \
+  echo 'jhipster:jhipster' |chpasswd && \
+  mkdir /home/jhipster/app && \
+  export DEBIAN_FRONTEND=noninteractive && \
+  export TZ=Europe\Paris && \
```

Comment:
Do we really need to set TZ.
```suggestion
```

Reply:
I put it there to avoid any issue related to the timezone in the container. Having it here doesn't make any difference.